src/Integration/WPF/RelayCommand.cs

```diff
@@ -49,12 +50,25 @@ public RelayCommand(Action execute, Func<bool> canExecute)
 
         public bool CanExecute()
         {
-            return this.canExecute?.Invoke() ?? true;
+            try
```

Comment:
Probably worth commenting that these methods can be called directly from XAML on the UI thread so we need to guard against unhandled exception.